src/fidesops/ops/util/saas_util.py

```diff
@@ -39,7 +39,7 @@ def load_config_with_replacement(
 ) -> Dict:
     """Loads the saas config from the yaml file and replaces any string with the given value"""
     yaml_str: str = load_yaml_as_string(filename).replace(
-        string_to_replace, replacement
+        string_to_replace, f'"{replacement}"'
```

Comment:
Is there any chance this causes a different problem? Would `'my "test" string'` replace to `'"my "test" string"'`?

Reply:
good question. I'm open to better ways to do this, it would be just wrapping if it's an integer type? or a redundant check here that the replacement is a "fides_key" type?

Our current usage of this by the connection template would fail before we got here, because we only let fides_keys have alphanumberic characters, '.', "_" or '-', it wouldn't allow the quotes to go through.

Reply:
Would this work?

```py
yaml_str: str = load_yaml_as_string(filename).replace(
        string_to_replace, str(replacement)
```

Reply:
No, it was already a string type here "111" so that is added to the yaml but when it is parsed, it comes out as an integer, so you need to add the quotes to force it to be a string.